src/sentry/api/paginator.py

```diff
@@ -330,3 +330,43 @@ def get_result(self, limit, cursor=None, count_hits=False):
             hits=min(len(self.scores), MAX_HITS_LIMIT) if count_hits else None,
             max_hits=MAX_HITS_LIMIT if count_hits else None,
         )
+
+
+class SnubaOffsetPaginator(object):
```

Comment:
I mentioned this in the other pull request and I'll mention it again here, this isn't specific to any particular Snuba API — can we just call this `OffsetPaginator` instead?

Reply:
there already is something called `OffsetPaginator` https://github.com/getsentry/sentry/blob/4583fa6280f46a6ef8475351b1b5cc5604fd2e36/src/sentry/api/paginator.py#L197

`GenericOffsetPaginator`? open to suggestions

Reply:
Dang. I don't have any better ideas offhand, I think`GenericOffsetPaginator` makes sense to me.

Reply:
done